handlers/aws/handler.py

```diff
@@ -202,87 +204,148 @@ def lambda_handler(lambda_event: dict[str, Any], lambda_context: context_.Contex
 
             return ret
 
-    if trigger_type == "sqs":
-        previous_sqs_record = 0
-        for es_event, last_ending_offset, current_sqs_record in _handle_sqs_event(
-            config, lambda_event, is_continuation_of_cloudwatch_logs
-        ):
-            shared_logger.debug("es_event", extra={"es_event": es_event})
-
-            composite_shipper.send(es_event)
-            sent_event += 1
+    if trigger_type == "s3-sqs" or trigger_type == "sqs":
+        composite_shipper_cache: dict[str, CompositeShipper] = {}
 
-            if current_sqs_record > previous_sqs_record:
-                sqs_record = lambda_event["Records"][previous_sqs_record]
-                delete_sqs_record(trigger_event_source_arn, sqs_record["receiptHandle"])
+        def event_processing_callback(
```

Comment:
Pretty minor: I find the term "callback" in the name `event_processing_callback()` a little distracting, my mind keeps asking "where is it used as callback function?".

Reply:
you are right, I will remove the suffix